core/src/test/java/org/infinispan/xsite/GlobalXSiteAdminOpsTest.java

```diff
@@ -0,0 +1,427 @@
+
+      //check state and data
+      assertSiteStatus(0, 0, null, 1, XSiteAdminOperations.ONLINE);
+      assertSiteStatus(0, 0, caches.get(CacheType.BACKUP_TO_SITE_1_AND_2).cacheName(), 1, XSiteAdminOperations.ONLINE);
+      assertSiteStatus(0, 0, caches.get(CacheType.BACKUP_TO_SITE_1_AND_2).cacheName(), 2, XSiteAdminOperations.OFFLINE);
+      assertSiteStatus(0, 0, caches.get(CacheType.BACKUP_TO_SITE_2).cacheName(), 2, XSiteAdminOperations.OFFLINE);
+
+      assertCacheEmpty(1, 0, null);
+      assertCacheEmpty(1, 0, caches.get(CacheType.BACKUP_TO_SITE_1_AND_2).backupCacheName());
+      assertCacheEmpty(2, 0, caches.get(CacheType.BACKUP_TO_SITE_1_AND_2).backupCacheName());
+      assertCacheEmpty(2, 0, caches.get(CacheType.BACKUP_TO_SITE_2).backupCacheName());
+
+      blockingTransportList.forEach(BlockingTransport::unblockCommands);
+   }
+
+   @AfterMethod(alwaysRun = true)
+   public void resetStatusAfterMethod() {
+      setSitesStatus(true);
+      getBlockingTransport(0, false).forEach(BlockingTransport::unblockCommands);
+   }
+
+   @Override
+   protected ConfigurationBuilder defaultConfigurationForSite(int siteIndex) {
+      if (siteIndex == 0) {
+         //default cache will backup to site_1
+         ConfigurationBuilder builder = newConfiguration();
+         builder.sites().addBackup().site(siteName(1)).strategy(BackupConfiguration.BackupStrategy.SYNC);
+         return builder;
+      } else {
+         return newConfiguration();
+      }
+   }
+
+   @Override
+   protected int defaultNumberOfSites() {
+      return 3;
+   }
+
+   protected void createCachesConfiguration() {
+      caches.put(CacheType.BACKUP_TO_SITE_2, new CacheConfiguration("cache2") {
+
+         @Override
+         public void defineConfiguration() {
+            ConfigurationBuilder builder = newConfiguration();
+            builder.sites().addBackup().site(siteName(2)).strategy(BackupConfiguration.BackupStrategy.SYNC);
+            defineInSite(site(0), cacheName(), builder.build());
+            defineInSite(site(2), backupCacheName(), newConfiguration().build());
+         }
+      });
+      caches.put(CacheType.BACKUP_TO_SITE_1_AND_2, new CacheConfiguration("cache3") {
+
+         @Override
+         public void defineConfiguration() {
+            ConfigurationBuilder builder = newConfiguration();
+            builder.sites().addBackup().site(siteName(1)).strategy(BackupConfiguration.BackupStrategy.SYNC);
+            builder.sites().addBackup().site(siteName(2)).strategy(BackupConfiguration.BackupStrategy.SYNC);
+            defineInSite(site(0), cacheName(), builder.build());
+            defineInSite(site(1), backupCacheName(), newConfiguration().build());
+            defineInSite(site(2), backupCacheName(), newConfiguration().build());
+         }
+      });
+      caches.put(CacheType.NO_BACKUP, new CacheConfiguration("cache4") {
```

Comment:
I think it would be more confusing. in this test, the cache name is the same in the all the sites, but in ```GlobalXSiteAdminOpsWithBackupForTest```, we have a different cache name in the backup sites.